cluster/gce/container-linux/configure-helper.sh

```diff
@@ -266,6 +266,12 @@ EOF
 multizone = ${MULTIZONE}
 EOF
   fi
+  if [[ -n "${GCE_ALPHA_FEATURES:-}" ]]; then
```

Comment:
It's used to control what GCE alpha features are enabled, so that we can gate certain features in the `pkg/cloudprovider/providers/gce`. See #50467 for the context.